<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rocket Ground Station</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icone.qrc">
    <normaloff>:/iconeWindows/icone/antenne-satellite.png</normaloff>:/iconeWindows/icone/antenne-satellite.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QWidget" name="telemetryPanel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="flightStatusLabel">
            <property name="text">
             <string>Flight Statut : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="gnssStatusLabel">
            <property name="text">
             <string>Statut GNSS : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="crcCheckLabel">
            <property name="text">
             <string>CrcCheck :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="latitudeLabel">
            <property name="text">
             <string>Latitude : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="longitudeLabel">
            <property name="text">
             <string>Longitude : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="altitudeLabel">
            <property name="text">
             <string>Altitude : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pressureLabel">
            <property name="text">
             <string>Pression : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="temperatureLabel">
            <property name="text">
             <string>Température : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="accZLabel">
            <property name="text">
             <string>Accélération Z : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="accXLabel">
            <property name="text">
             <string>Accélération X : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="accYLabel">
            <property name="text">
             <string>Accélération Y : </string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_map">
          <attribute name="title">
           <string>Map</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="MapZone" name="mapzone" native="true">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_data">
          <attribute name="title">
           <string>Data</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="console">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>200</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="tabChangesFocus">
          <bool>false</bool>
         </property>
         <property name="documentTitle">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="plainText">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClean"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionConfigure"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeSerial/icone/usb.png</normaloff>:/iconeSerial/icone/usb.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeSerial/icone/disconnect.png</normaloff>:/iconeSerial/icone/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeWindows/icone/remove.png</normaloff>:/iconeWindows/icone/remove.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeSerial/icone/settings.png</normaloff>:/iconeSerial/icone/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeWindows/icone/question.png</normaloff>:/iconeWindows/icone/question.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeWindows/icone/Qt_logo_2016.svg.png</normaloff>:/iconeWindows/icone/Qt_logo_2016.svg.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="icon">
    <iconset resource="icone.qrc">
     <normaloff>:/iconeWindows/icone/broom.png</normaloff>:/iconeWindows/icone/broom.png</iconset>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MapZone</class>
   <extends>QWidget</extends>
   <header location="global">MapZone.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icone.qrc"/>
 </resources>
 <connections/>
</ui>
